### ISSUE TITLE
kws_streaming On-Device training

### ISSUE THREAD

**Issue by StuartIanNaylor** [google-research/google-research#1422]
@rybakov just wondering if you have or maybe could implement an on-device training scheme as in
https://www.tensorflow.org/lite/examples/on_device_training/overview

I am trying to work out how it could be implemented with the steaming subclassing, but not doing well :)

**Comment by rybakov** [comment]
I would suggest to follow an [example](https://www.tensorflow.org/lite/examples/on_device_training/overview) and use TF V2 in kws_streaming, this way the above example should be compatible with kws_streaming and the reaming question would be about TFLite supported ops.

**Comment by StuartIanNaylor** [comment]
Yeah not that sure how with the streaming subclassing, but yeah have read that article and will give it a go.
